src/Plutus/Certification/API/Routes.hs

```diff
@@ -446,8 +447,34 @@ instance ToSchema DAppBody where
     return $ NamedSchema (Just "DAppBody") $ profileSchema
         & properties . at "githubToken" ?~ apiGitHubAccessTokenSchema
 
-instance ToSchema ProfileBody
+-- | A phantom type for website for swagger schema
+data Website
 
+instance ToSchema Website where
+  declareNamedSchema _ = do
+    return $ NamedSchema (Just "BaseUrl") $ mempty
+      & type_ ?~ SwaggerString
+      & SL.pattern ?~ ghAccessTokenPattern
+
+baseUrlPattern :: Pattern
+baseUrlPattern = "^[(http(s)?):\\/\\/(www\\.)?a-zA-Z0-9@:%._\\+~#=]{2,256}\\.[a-z]{2,6}\\b([-a-zA-Z0-9@:%_\\+.~#?&//=]*)$"
+
+instance ToSchema ProfileBody where
+   declareNamedSchema _ = do
+    textSchema <- declareSchemaRef (Proxy :: Proxy Text)
+    dappSchema <- declareSchemaRef (Proxy :: Proxy DAppBody)
+    urlSchema <- declareSchemaRef (Proxy :: Proxy Website)
+    return $ NamedSchema (Just "Profile") $ mempty
+      & type_ ?~ SwaggerObject
+      & properties .~
+          [ ("dapp", dappSchema)
+          , ("website", urlSchema)
+          , ("vendor", textSchema)
+          , ("twitter", textSchema)
```

Comment:
Couldn't you also use urlSchema for twitter and linkedin?

Reply:
Good point. Yeah maybe this schéma for Twitter handle would be perfect

Reply:
done